2_Growth-Maturity-Decline.md

```diff
@@ -1,15 +1,33 @@
 # Metric: Growth-Maturity-Decline
 
 
+## Issue Resolution
  Activity Metric | Description
  --- | ---
- Tickets Closed / Tickets Opened | Total number of tickets closed per tickets opened over time
- Bug Time to Close | Time from bug opened to bug closed over time
- Time to Merge | Time from submission to merge or abandon over time
- Review Efficiency | Number of merged patches / Number of abandoned patches over time
- Number of New Contributors | Number of first time contributors over time
- Community Growth | Number of new developers over time
- Community Toxicity | Toxic behavior and language over time
+ Open issues | Total number of open issues.
+ Closed issues | Total number of closed issues.
+ Resolution efficiency | Number of closed issues / number of abandoned issues.
+ Average & median time of open issues | The average & median amount of time issues remain open.
+ Average & median time of first response | The average & median amount of time until the first response to an issue is made.
+ Distribution of open issues over time | Proportional frequency of open issue time duration.
+ Distribution of issue resolution over time | Proportional frequency of closed issue time duration.
+
+## Code Development
+ Activity Metric | Description
+ --- | ---
+ Number of commits | Total number of commits.
+ Number of lines changed | Total number of lines of code that have been changed.
+ Average & median time to merge code | The average & median amount of time it takes to get code merged upstream. 
+ Distribution of time to merge | Proportional frequency of code merge to upstream time duration.
+ Review efficiency | Number of merged patches / number of abandoned patches.
+
+## Project Communication
+ Activity Metric | Description
+ --- | ---
+ Total communications | Total number of posts made to communication platforms.
+ Average & median communication response time | The average & median time taken for communications to be responded to.
+ Average & median communication responses per thread | The average & median number of responses that are made to each communication thread or topic.

```

Comment:
How is this different to the previous one? Can we measure response time for messages that are not related (i.e. in the same thread) - asking for previous criteria because I imagined we had to measure on a per thread basis?

Reply:
> Average & median communication responses per thread

This metric doesn't relate to time, it counts the number of actual posts that are made in response to a thread. E.g. if two people make one post each in response to someone's email, then this number would be two.